app/src/main/java/com/willowtree/vocable/presets/PresetsFragment.kt

```diff
@@ -303,6 +304,19 @@ class PresetsFragment : BaseFragment() {
             }
             numPages = ceil(phrases.size / maxPhrases.toDouble()).toInt()
             notifyDataSetChanged()
+
+            if (phrasesAdapter.numPages == 1) {
```

Comment:
You can simplify this by just calling `setPagingButtonsEnabled(phrasesAdapter.numPages > 1)`